tuf/api/metadata.py

```diff
@@ -834,15 +800,10 @@ def update(self, snapshot_meta: MetaFile) -> None:
 class Snapshot(Signed):
     """A container for the signed part of snapshot metadata.
 
-    Attributes:
-        meta: A dictionary that contains information about targets metadata::
-
-            {
-                'targets.json': <MetaFile INSTANCE>,
-                '<DELEGATED TARGETS ROLE 1>.json': <MetaFile INSTANCE>,
-                '<DELEGATED TARGETS ROLE 2>.json': <MetaFile INSTANCE>,
-            }
+    Snapshot contains information about all target Metadata files.
 
+    Attributes:
+        meta: A dictionary of target metadata filenames to MetaFiles.
```

Comment:
```suggestion
        meta: A dictionary of target metadata filenames to MetaFile objects.
```
Wanted to preserve the keyword MetaFile